test/ca-app-context-test.html

```diff
@@ -0,0 +1,97 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>CA APP CONTEXT | Web Lab Common UI</title>
+    <script src="/web-component-tester/browser.js"></script>
+    <script src="/bower_components/system.js/dist/system.js"></script>
+    <script src="/components/config/index.js"></script>
+  </head>
+  <body>
+    <test-fixture id="app-context">
+      <template>
+        <ca-app-context></ca-app-context>
+      </template>
+    </test-fixture>
+    <script>
+        suite('<ca-app-dialog>', () => {
+            let appContext;
+
+            setup(done => {
+                SystemJS.import('ca-app-context.js').then(() => {
+                    appContext = fixture('app-context');
+                    done();
+                });
+            });
+
+            test('should be creatable', () => {
+                // Check the element has been created with the correct tag
+                expect(appContext).to.be.defined;
+                expect(appContext.tagName).to.equal('CA-APP-CONTEXT');
+            });
+
+            test('should listen to config event and update element according when value is present', function() {
+                // setup
+                var testElement = document.createElement('div');
+                var event = new CustomEvent('ca-config-component', {
+                  'detail': {
+                      config: [
+                          { name: 'rpi', localName: 'data-rpi', type: 'attribute' },
+                      ],
+                      component: testElement
+                  },
+                  'bubbles': true
+                });
+
+                // act
+                appContext.setAttribute('data-rpi', true);
+                testElement.dispatchEvent(event);
+
+                // verify
+                expect(appContext.getAttribute('data-rpi')).to.be.defined;
```

Comment:
I agree that this is a direct port, but it needs a TODO as it's not actually testing anything useful. It ought to be checking the attribute on the testElement instead.